kolibri/core/assets/src/validators.js

```diff
@@ -25,3 +25,195 @@ export function validateLearningActivity(arr) {
   const isValidLearningActivity = v => Object.values(LearningActivities).includes(v);
   return arr.length > 0 && arr.every(isValidLearningActivity);
 }
+export function objectPropValidator( options ){
```

Comment:
Would be good to have a test suite to demonstrate the validation happening here.